openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -125,17 +135,34 @@ def _update_tool_with_data(self, tool, data):
         original_subset = tool.GetData("openpype.subset")
         subset = data["subset"]
         if original_subset != subset:
-            # Subset change detected
-            # Update output filepath
-            workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
-            filename = f"{subset}..exr"
-            filepath = os.path.join(workdir, "render", subset, filename)
-            tool["Clip"] = filepath
-
-            # Rename tool
-            if tool.Name != subset:
-                print(f"Renaming {tool.Name} -> {subset}")
-                tool.SetAttrs({"TOOLS_Name": subset})
+            self._configure_saver_tool(data, tool, subset)
+
+    def _configure_saver_tool(self, data, tool, subset):
+        formatting_data = deepcopy(data)
+        self.log.warning(pformat(formatting_data))
+
+        # Subset change detected
+        workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
+        formatting_data.update({
+            "workdir": workdir.replace("\\", "/"),
+            "ext": "exr"
+        })
+
+        # build file path to render
+        filepath = self.temp_rendering_path_template.format(
+            **formatting_data)
+
+        # create directory
+        if not os.path.isdir(os.path.dirname(filepath)):
+            self.log.warning("Path does not exist! I am creating it.")
```

Comment:
Logged message is very unclear. I would not auto-create these folders really.
I believe the Fusion saver has a setting to auto create directories if needed.

If we do want to create the folders then I'd at least log what path is being created.

Reply:
Yep, Fusions savers can create the folders when it starts to render. You even have a validator that checks that the `create folder` is checked